features/FxA-97-password-strength/README.md

```diff
@@ -73,7 +73,48 @@ When the user enters any characters, the instructions change to become:
 When the user enters a password that is on our blocklist, the UI will behave as if the following error will be shown. The user is still able to submit the form if unchanged. If the user modifies the password, it starts the process over again, and is only submitted if the password is strong enough.
 * This is a common password; please consider another one.  <u>More info</u>.
 
-### Unresolved questions and risks
+## Results
 
-[comment]: # ( What parts of the design are still TBD?)
-More info page copy.
+We have deployed this feature to 10% of FxA users.
+Before showing the "weak" password tooltip
+we gathered some metrics about the bloomfilter.
+That data can be found in the "Metrics" section
+of the document.
+
+Since July 2016 and introducing new password prompts the following changed:
+
+* Passwords that are just letters and numbers increased by 2%.
+* Passwords that are too common (aka BLOOMFILTER_HIT) remained unchanged (1.7%).
+* Bloom filter misses increased by 10%.
+* Successful signups as a % of submissions decreased by 10%.
```

Comment:
One request, I think it would be worth noting the actual values as well as the amount by which they changed.